src/main.ts

```diff
@@ -80,50 +80,81 @@ export interface LoadBalancerAccessibility {
 export interface FargateTaskProps {
   // The Fargate task definition
   readonly task: ecs.FargateTaskDefinition;
+
   /**
    * The internal ELB listener
    * @default - no internal listener
    */
   readonly internal?: LoadBalancerAccessibility;
+
   /**
    * The external ELB listener
    * @default - no external listener
    */
   readonly external?: LoadBalancerAccessibility;
+
   /**
    * desired number of tasks for the service
    * @default 1
    */
   readonly desiredCount?: number;
+
   /**
    * service autoscaling policy
    * @default - { maxCapacity: 10, targetCpuUtilization: 50, requestsPerTarget: 1000 }
    */
   readonly scalingPolicy?: ServiceScalingPolicy;
+
   /**
    * Customized capacity provider strategy
    */
   readonly capacityProviderStrategy?: ecs.CapacityProviderStrategy[];
+
   /**
    * health check from elbv2 target group
   */
   readonly healthCheck?: elbv2.HealthCheck;
+
   /**
    * The target group protocol for NLB. For ALB, this option will be ignored and always set to HTTP.
    *
    * @default - TCP
    */
   readonly protocol?: elbv2.Protocol;
+
   /**
    * The protocol version to use.
    */
   readonly protocolVersion?: elbv2.ApplicationProtocolVersion;
+
   /**
    * The serviceName.
    *
    * @default - auto-generated
    */
   readonly serviceName?: string;
+
+  /**
+   * Set/replace the target group's health check.
+   * @default
+   * health check interval seconds is 30 seconds.
+   * health check path is '/'.
+  */
+
+  readonly healthCheckConfigure?: elbv2.HealthCheck;
+  /**
+   * The maximum number of tasks, specified as a percentage of the Amazon ECS service's DesiredCount value,
+   * that can run in a service during a deployment.
+   * @default- 100 if daemon, otherwise 200
```

Comment:
```suggestion
   * @default 200
```

We don't run daemon mode in this construct so the default should be 200.